DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -90,10 +90,6 @@ public void GetGroupCoursesForCentre_returns_expected_courses()
             {
                 1,
                 2,
-                21,
-                22,
-                23,
-                24,

```

Comment:
It would be good to have a couple of tests that courses with DefaultContentTypeID = 4 are discounted. I guess these are such courses?
I wouldn't worry about adding a test for every method you've changed, but even if it's just for the groups data service and activity data service, and maybe one for course data service.

Reply:
Those are the courses that vanished when filtering out ID 4, yeah. I didn't add a new test for each method but did update several tests where failures were introduced. I'll have a look at what ones haven't been covered.

Reply:
With a change I've done to activity data service tests I think all of it, groups and course are covered.